buildSrc/src/main/resources/resource_file_template.txt

```diff
@@ -17,7 +17,5 @@
   limitations under the License.
 -->
 <resources>
-{% for animation in data %}
-	<integer name="{{animation.name}}">{{animation.value}}</integer>
-{% endfor %}
+{{content}}
```

Comment:
the git diff is a bit confusing, but basically this is the construct of any resource files. the ``{{content}}` bit is replaced in code